Kitodo/src/main/java/org/kitodo/production/forms/dataeditor/PaginationPanel.java

```diff
@@ -0,0 +1,250 @@
+/*
+ * (c) Kitodo. Key to digital objects e. V. <contact@kitodo.org>
+ *
+ * This file is part of the Kitodo project.
+ *
+ * It is licensed under GNU General Public License version 3 or later.
+ *
+ * For the full copyright and license information, please read the
+ * GPL3-License.txt file that was distributed with this source code.
+ */
+
+package org.kitodo.production.forms.dataeditor;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import javax.faces.model.SelectItem;
+
+import org.kitodo.api.dataformat.MediaUnit;
+import org.kitodo.config.ConfigCore;
+import org.kitodo.config.enums.ParameterCore;
+import org.kitodo.production.helper.Helper;
+import org.kitodo.production.helper.metadata.pagination.Paginator;
+import org.kitodo.production.helper.metadata.pagination.PaginatorMode;
+import org.kitodo.production.helper.metadata.pagination.PaginatorType;
+import org.kitodo.production.helper.metadata.pagination.RomanNumeral;
+import org.kitodo.production.services.ServiceManager;
+
+/**
+ * Backing bean for the pagination panel.
+ */
+public class PaginationPanel {
+
+    private DataEditorForm dataEditor;
+    private boolean fictitiousCheckboxChecked = false;
+    private int newPagesCountValue = 0;
+    private List<SelectItem> paginationSelectionItems;
+    private List<Integer> paginationSelectionSelectedItems = new ArrayList<>();
+    private String paginationStartValueValue = "1";
+    private List<SelectItem> paginationTypeSelectItems;
+    private PaginatorType paginationTypeSelectSelectedItem = PaginatorType.ARABIC;
+    private List<IllustratedSelectItem> selectPaginationModeItems;
+    private PaginatorMode selectPaginationModeSelectedItem = PaginatorMode.PAGES;
+    private List<SelectItem> selectPaginationScopeItems;
+    private Boolean selectPaginationScopeSelectedItem = Boolean.TRUE;
+
+    /**
+     * Constructor.
+     *
+     * @param dataEditor DataEditorForm instance
+     */
+    public PaginationPanel(DataEditorForm dataEditor) {
+        this.dataEditor = dataEditor;
+        preparePaginationTypeSelectItems();
+        prepareSelectPaginationModeItems();
+        prepareSelectPaginationScopeItems();
+    }
+
+    /**
+     * This method is invoked if the create pagination button is clicked.
+     */
+    public void createPaginationButtonClick() {
+        ServiceManager.getFileService().searchForMedia(dataEditor.getProcess(), dataEditor.getWorkpiece());
+        Paginator paginator = new Paginator(metsEditorDefaultPagination(1));
+        List<MediaUnit> mediaUnits = dataEditor.getWorkpiece().getMediaUnits();
+        for (int i = 1; i < mediaUnits.size(); i++) {
+            MediaUnit mediaUnit = mediaUnits.get(i);
+            mediaUnit.setOrder(i);
+            mediaUnit.setOrderlabel(paginator.next());
+        }
+    }
+
+    /**
+     * This method is invoked if the generate dummy images button is clicked.
+     */
+    public void generateDummyImagesButtonClick() {
```

Comment:
I can't answer for sure whether generating dummy images might be required again in the future since the workflow in Production 3 is more flexible than in 2 and allows performing workflow tasks in some order that might not have been used in prior versions, so I would rather keep this function for now.